src/mail/package.json

```diff
@@ -1,23 +1,22 @@
 {
-    "name": "mail_mjml",
-    "version": "2.5.1",
-    "description": "An util to generate html and text jinja templates from mjml templates",
-    "type": "module",
-    "dependencies": {
-      "@html-to/text-cli": "0.5.4",
-      "mjml": "4.15.3"
-    },
-    "private": true,
-    "scripts": {
-      "build-mjml-to-html": "./bin/mjml-to-html",
-      "build-html-to-plain-text": "./bin/html-to-plain-text",
-      "build": "yarn build-mjml-to-html; yarn build-html-to-plain-text;"
-    },
-    "volta": {
-      "node": "16.15.1"
-    },
-    "repository": "https://github.com/openfun/mork",
-    "author": "France Université Numérique",
-    "license": "MIT"
-  }
-  
\ No newline at end of file
+  "name": "mail_mjml",
+  "version": "2.5.1",
```

Comment:
IMO the version should be sync with backend application version

Reply:
Oh good point!